balta/src/main/scala/za/co/absa/db/balta/classes/QueryResultRow.scala

```diff
@@ -35,7 +36,10 @@ class QueryResultRow private[classes](val rowNumber: Int,
                                       private val columnNames: FieldNames) {
 
   def columnCount: Int = fields.length
-  def columnNumber(columnLabel: String): Int = columnNames(columnLabel.toLowerCase)
+  def columnNumber(columnLabel: String): Int = {
```

Comment:
why `Label` terminology?

Reply:
Original terminology from JDBC

Reply:
but it's column name right?

Reply:
Hmm I read here that it's this: https://bugs.mysql.com/bug.php?id=35610

so `alias` or `column name`. Not sure if in our case it's always only column name?

Reply:
Look at the `ResultSet` class methods.

Reply:
Hmm I still don't fully know the answer if you think deeper into it :D 

https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html 

E.g. 

* columnLabel - the label for the column specified with the SQL AS clause. If the SQL AS clause was not specified, then the label is the name of the column
* Column names used as input to getter methods are case insensitive. When a getter method is called with a column name and several columns have the same name, the value of the first matching column will be returned. The column name option is designed to be used when column names are used in the SQL query that generated the result set. For columns that are NOT explicitly named in the query, it is best to use column numbers. If column names are used, the programmer should take care to guarantee that they uniquely refer to the intended columns, which can be assured with the SQL AS clause.

Okay, we can follow it. But maybe at least document it super briefly? That this terminology is coming from X with some link perhaps etc

Reply:
(Basically, if we are using an underlying technology A that is using special terminology B for something that uses X and Y, and X is known thing - in our case column name - and we are not using Y, then it makes sense not to follow A's way and use B, just use X)

Reply:
Can you please suggest the documentation text. I am not sure I follow.